TerminalDocs/customize-settings/profile-appearance.md

```diff
@@ -414,7 +414,7 @@ This sets the visibility of the scrollbar.
 
 **Necessity:** Optional
 
-**Accepts:** `"visible"`, `"hidden"`
+**Accepts:** `"visible"`, `"hidden"`, `"always"`
```

Comment:
Hi, I come from https://learn.microsoft.com/en-us/windows/terminal/customize-settings/profile-appearance#scrollbar-visibility , finding that "always" is not available yet in stable v1.15.3465.0

Would you please clarify in which version "always" would take effect? Thanks.

Reply:
I was told that the "always" value will be in the 1.17 release. Looks like it was not successfully removed here - apologies for the confusion!